GoogleForCreators/web-stories-wp
# Fix TinyMCE classic editor integration

## Context



While looking into e2e tests I noticed an issue with the TinyMCE button integration in the classic editor, which was causing tons of JavaScript errors.

## Summary



Fixes console errors to ensure the TinyMCE button works again as expected.

## Relevant Technical Choices



Updated `@wordpress/data` usage according to best practices. Uses functions that are available since WordPress 5.7, so compatibility is given.

Also simplified the code a little bit to make it easier to maintain.

## To-do



- Maybe some follow-up improvements
- Fix and re-enable e2e tests in follow-up tickets

## User-facing changes



TinyMCE integration should be 

## Testing Instructions






This PR can be tested by following these steps:

1. Activate Classic Editor plugin
2. Create a new blog post
3. See Web Stories button
4. Verify you're able to use that button to insert a shortcode.


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #
